src/types/video.ts

```diff
@@ -61,4 +61,34 @@ export type UnprocessedVideoInfo = {
    *  The height of the video
    */
   vHeight: number
+  /**
+   *  Video hash
+   */
+  hash: string
```

Comment:
I saw this added here, but we are already using the hash as the actual object key. Does this need to be duplicated? Will comment about the alternative on the `processVideoChunksAndTelemetry` changes.